test/interface.jl

```diff
@@ -847,10 +847,19 @@ end
     for i in 1:length(bs), j in 1:length(bs)
         if i == j
             @test bs[i] == deepcopy(bs[j])
-            @test inv(bs[i]) == inv(deepcopy(bs[j]))
+            @test inverse(bs[i]) == inverse(deepcopy(bs[j]))
         else
             @test bs[i] != bs[j]
         end
     end
 end
 
+@testset "deprecations" begin
+    b = Bijectors.Exp()
+    x = 0.3
+
+    @test let r = forward(b, x)
+        (r.rv, r.logabsdetjac) == with_logabsdet_jacobian(b, x)
+    end
```

Comment:
Maybe use something like
```suggestion
    @test @test_deprecated(forward(b, x)) == NamedTuple{(:rv, :logabsdetjac)}(with_logabsdet_jacobian(b, x))
```
to also check that the method is deprecated and that it actually returns a named tuple with the two fields.